### DISCUSSION TITLE
Invalid URL in field "id" (in "itemListElement.item") for a Docusaurus website

### DISCUSSION THREAD

**Discussion by chengtie** [facebook/docusaurus#9549] [Q&A]
(* StackOverflow: https://stackoverflow.com/questions/77480300 *)

We have a website made with Docusaurus "2.0.0-beta.18". I'm improving its SEO by inspecting its pages in Google Search Console. For most of its page, I realize that there are 2 non-critical issues. Here is a screenshot of live test of www.10studio.tech/docs/formula-editor:

[![enter image description here][1]][1] 

And the corresponding HTML in the tested page is

```
<ul class="breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList"><li class="breadcrumbs__item"><a class="breadcrumbs__link" href="/">🏠</a></li><li itemscope="" itemprop="itemListElement" itemtype="https://schema.org/ListItem" class="breadcrumbs__item"><span class="breadcrumbs__link" itemprop="item name">Formula Editor</span><meta itemprop="position" content="1"></li><li itemscope="" itemprop="itemListElement" itemtype="https://schema.org/ListItem" class="breadcrumbs__item breadcrumbs__item--active"><span class="breadcrumbs__link" itemprop="item name">Tool</span><meta itemprop="position" content="2"></li></ul>
```

Does anyone know how to fix this?

  [1]: https://i.stack.imgur.com/zfd4c.png

**Comment by Josh-Cena** [comment]
Hi, we don't support legacy versions, and I believe this has been fixed in a recent version. If you want to invest time fixing itself, try [swizzling](https://docusaurus.io/docs/swizzling).
